src/core/thread/network_data.cpp

```diff
@@ -217,6 +217,10 @@ otError NetworkData::GetNextOnMeshPrefix(Iterator &aIterator, uint16_t aRloc16,
                     aConfig.mOnMesh         = borderRouterEntry->IsOnMesh();
                     aConfig.mStable         = borderRouter->IsStable();
                     aConfig.mRloc16         = borderRouterEntry->GetRloc();
+                    aConfig.mNdDns          = borderRouterEntry->IsNdDns();
+#if OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2
```

Comment:
I think we should keep this even for Thread 1.1. Being able to get the Domain Prefix flag may be useful.

Reply:
updated.